app/src/main/java/org/fossasia/openevent/general/ticket/TicketsViewModel.kt

```diff
@@ -37,7 +41,11 @@ class TicketsViewModel(
     val event: LiveData<Event> = mutableEvent
     private val mutableDiscountCodes = MutableLiveData<DiscountCode>()
     val discountCode: LiveData<DiscountCode> = mutableDiscountCodes
+    private val mutableTaxInfo = MutableLiveData<Tax>()
+    val taxInfo: LiveData<Tax> = mutableTaxInfo
     var appliedDiscountCode: DiscountCode? = null
+    var tax: Tax? = null
```

Comment:
Remove this variable. Totally unnecessary

Reply:
removed